packages/shared/lib/sdk/sync.ts

```diff
@@ -427,26 +427,21 @@ export class NangoAction {
 
             const proxyConfig = this.proxyConfig(config);
 
-            const { response, activityLogs: activityLogs } = await proxyService.route(proxyConfig, {
-                existingActivityLogId: this.activityLogId as number,
+            const { response, logs } = await proxyService.route(proxyConfig, {
+                existingActivityLogId: this.activityLogId as string,
                 connection,
                 provider: this.provider as string
             });
 
-            if (activityLogs) {
-                // Save buffered logs
-                for (const log of activityLogs) {
+            // We batch save, since we have buffered the createdAt it shouldn't impact order
+            await Promise.all(
+                logs.map(async (log) => {
                     if (log.level === 'debug') {
-                        continue;
+                        return;
                     }
-
-                    if (!this.dryRun) {
```

Comment:
Where is dryRun being handled now?

Reply:
in the logCtx creation the dryRun parameter. That's mostly why I embedded it so it's transparent and one less param to carry around